src/main/java/com/datasonnet/header/Header.java

```diff
@@ -87,6 +87,12 @@ public void setVersion(String version) {
         this.version = version;
     }
 
+    public boolean isPreserveOrder() {
+        Map output = getOrEmpty(dataFormatParameters, Header.DATASONNET_OUTPUT);
+        boolean preserveOrder = new Boolean(output.getOrDefault(Header.DATASONNET_OUTPUT_PRESERVE_ORDER, "true").toString());
```

Comment:
The docs should be updated with info on this in the same PR

Reply:
Good catch, will do.